src/krisi/evaluate/library/metric_wrappers.py

```diff
@@ -97,7 +97,9 @@ def y_label_imbalance_ratio(
 ) -> float:
     label_frequencies = y.value_counts(normalize=True).dropna()
     if len(label_frequencies) == 2:
-        return label_frequencies[pos_label]
+        return min(
```

Comment:
so we don't ned up with values above 1, while it's still interpretable

Reply:
it can be either the positive or the negative class, 1-ratio_of_pos_label should be == the negative class' frequency, so we get the smaller number of the two, so we don't end up with a value larger than 1